### ISSUE TITLE
Shouldn't be using --socket=session-bus

### ISSUE THREAD

**Issue by TingPing** [flathub/work.openpaper.Paperwork#1]
Sorry I didn't catch this at review time but: 

```json
        "--talk-name=org.freedesktop.Notifications",
        "--talk-name=org.freedesktop.DBus",
        "--socket=session-bus",
        "--own-name=work.openpaper.paperwork"
```

You can always talk to `org.freedesktop.DBus` and `session-bus` completely grants access to everything so these don't make sense.

I didn't want to push changing anything without testing it so I thought I'd point it out.

**Comment by jflesch** [comment]
Good point. I'll have a try next week. In that case, I think I will also have to add `--talk-name=org.freedesktop.FileManager1`

**Comment by hadess** [comment]
Was fixed in 6fe1fd90115e5665ba5081776ed0c5ddac6df673

**Comment by jflesch** [comment]
Good point, thanks. I'm closing this issue.
